### ISSUE TITLE
Reader hardware support

### ISSUE THREAD

**Issue by mikekelly** [mclear/OMNI-Ring#14]
Hi, I can see from the docs that I need a "PCSC compatible NFC reader" but is there anything more specific I should be looking for? I'm having trouble getting an HID Omnikey 5022CL to work. Is that compatible?

are you able to provide any quick pointers here please @JohnMcLear? :)

**Comment by JohnMcLear** [comment]
Sorry @mikekelly this isn't something I'm involved with right now but I'd recommend Google searching pcsc Linux devices and seeing which makes and models come up.

**Comment by mikekelly** [comment]
ok fair enough thanks @JohnMcLear. Would be cool to chat some time btw, I could do with your advice more broadly! :)

**Comment by JohnMcLear** [comment]
> ok fair enough thanks @JohnMcLear. Would be cool to chat some time btw, I could do with your advice more broadly! :)

Yeah I'm happy to chat but I might be useless, email me with a good way to contact you.

**Comment by mikekelly** [comment]
Awesome thanks - ***@***.***

Sent via Superhuman iOS ***@***.***>
